static/app/bootstrap/renderSystemAlerts.tsx

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+
+import SystemAlerts from 'app/views/app/systemAlerts';
+
+export function renderSystemAlerts(container: string, props?: Record<string, any>) {
```

Comment:
Also seems like maybe this one isn't use anymore?

Reply:
Oh yeah good catch, forgot to remove these 👍